### PR TITLE
Add prerequisites to contributing docs

### PR DESCRIPTION
**Pull request by DanielMSchmidt**
While setting up the project I found these dependencies to the development machine to exist :)

### CODE REVIEW

**Comment by danieldreier** [comment]
👋 welcome!

**Comment by jsteinich** [comment]
We should probably mention the docker image that has all the dependencies bundled: `hashicorp.jfrog.io/docker/hashicorp/jsii-terraform` (Dockerfile in the repo)

**Comment by DanielMSchmidt** [comment]
That sounds like a great idea, maybe adding a wrapper shell script could help folks, like `./docker.sh package`, `./docker.sh integration`, etc

**Comment by DanielMSchmidt** [comment]
@jsteinich Great point, didn't notice the dockerfile previously. I added a note about it as well as the vscode dev containers so folks can choose :)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -88,7 +102,7 @@ For Python [examples](./examples/python), packages are referenced from `./dist`,
 
 For Java [examples](./examples/java), packages are referenced from `./dist`, there's no symlinking possible for live code updates. You'll have to explictly run `yarn package` to create new packages to be referenced in the pom.
 
-#### C#
```
Comment (jsteinich) [inline CONTRIBUTING.md:91]: Shouldn't remove this.

**Review by jsteinich — CHANGES_REQUESTED** [review]
Looks good besides a small accidental change.

**Review by jsteinich — APPROVED** [review]

**Pull request merged**